angular-ngrx-scss/src/app/app.module.ts

```diff
@@ -27,7 +29,13 @@ import { UserEffects } from './state/user';
       logOnly: environment.production,
       autoPause: true,
     }),
-    EffectsModule.forRoot([UserEffects, ProfileEffects, RepositoryEffects]),
+    EffectsModule.forRoot([
+      // DashboardEffects,
+      AuthEffects,
+      UserEffects,
+      ProfileEffects,
+      RepositoryEffects,
+    ]),
```

Comment:
is splitting the reducers out of the global declaration into a forFeature setup part of this PR?

Reply:
No, it's not. Everything right now is intended to be global.